src/extensions/averagewaveforms/AverageWaveformsView/AverageWaveformView.tsx

```diff
@@ -1,6 +1,6 @@
-import React, { FunctionComponent, useState } from 'react';
+import React, { FunctionComponent } from 'react';
 import { SortingSelection, SortingSelectionDispatch } from '../../extensionInterface';
-import AverageWaveformWidget from './AverageWaveformWidget';
+import WaveformWidget from './WaveformWidget';
```

Comment:
Renamed AverageWaveformWidget to WaveformWidget because it is also used by snippets view